scripts/parallellize_cargo_check.py

```diff
@@ -38,8 +38,9 @@
     print(os.environ)
     if "libafl_frida" in task:
         # DOCS_RS is needed for libafl_frida to build without auto-download feature
-        cargo_check = subprocess.check_output(
-            task, shell=True, text=True, env=dict(os.environ, DOCS_RS="1")
-        )
+        # cargo_check = subprocess.check_output(
```

Comment:
What happened to this check?

Reply:
https://github.com/AFLplusplus/LibAFL/actions/runs/11204078318/job/31142122206

we can keep it this way for now, once @s1341 finished to fix the problem we can open another pr to re-enable clippy for frida.

Reply:
I fear we'll just forget it. Can't we keep it broken?

Reply:
ok i don't mind